providers/implementations/rands/seed_src.c

```diff
@@ -206,6 +206,7 @@ static size_t seed_get_seed(void *vseed, unsigned char **pout,
                           adin, adin_len) != 0)
         return bytes_needed;
     OPENSSL_secure_clear_free(p, bytes_needed);
+    *pout = NULL;
```

Comment:
Would it be better to only set `pout` in the case the call to `seed_src_generate` succeeds?

Reply:
As you wish. I don't see any significant difference, though your approach looks a bit more safe.

Reply:
Done, thanks!